### ISSUE TITLE
Deobfuscate heapsnapshot and instruction sizes in gen_snapshot.

### ISSUE THREAD

**Issue by chingjun** [dart-lang/sdk#42752]
When obfuscation is enabled, gen_snapshot should write the unobfuscated version of the outputs of `--write-v8-snapshot-profile-to=` and `--print-instructions-sizes-to=`. The obfuscated version is not very useful.

Currently we are adding a script to deobfuscate the output files.

Thanks.

**Comment by vsmenon** [comment]
Do we need this for internal build work?

**Comment by chingjun** [comment]
Yes but it's not urgent. It's not critical and we have a workaround currently. cl/321853147
